rasa/nlu/test.py

```diff
@@ -664,9 +633,53 @@ def evaluate_intents(
         "precision": precision,
         "f1_score": f1,
         "accuracy": accuracy,
+        "errors": intent_errors,
     }
 
 
+def _calculate_report(
+    filename: Text,
+    output_directory: Optional[Text],
+    targets: Iterable[Any],
+    predictions: Iterable[Any],
+    report_as_dict: bool = False,
+    exclude_label: Optional[Text] = None,
+) -> Tuple[Union[Text, Dict], float, float, float, np.ndarray, List[Text]]:
+    from rasa.test import get_evaluation_metrics
```

Comment:
Do we have general guidelines or rules for when we do local imports?

Reply:
In this case it avoid a cyclic import and sklearn is a very heavy dependency which means the local import avoid long loading times of the CLI (or better: avoids even longer loading times). 

But no, we don't have them.